.gemini.yml

```diff
@@ -0,0 +1,14 @@
+rootUrl: http://60c4a4bf.ngrok.io
+gridUrl: http://ondemand.saucelabs.com/wd/hub
+windowSize: 1024x768
+system:
+  plugins:
+    sauce:
+      username:
+      accessKey:
+browsers:
+  chrome:
+    desiredCapabilities:
```

Comment:
Can we configure this to have a better test name? Currently it looks like this in Sauce Labs which isn't helpful: 
<img width="1189" alt="screenshot 2017-01-18 20 34 04" src="https://cloud.githubusercontent.com/assets/22905/22093823/a2d4308e-ddbd-11e6-92a9-6fdf5a888d63.png">

I think the `name` key does the trick.